ledger/participant-state/kvutils/tools/src/test/scala/com/daml/Replay.scala

```diff
@@ -17,14 +17,17 @@ import com.daml.lf.archive.{Decode, UniversalArchiveReader}
 import com.daml.lf.crypto
```

Comment:
It would make sense to move this class into a separate package, e.g. to under `com.daml.replay`.